org.eclipse.m2e.wtp.jaxrs/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.m2e.wtp.jaxrs;singleton:=true
-Bundle-Version: 1.6.0.qualifier
```

Comment:
Why this change here? Only modifies bundles should have their version bumped (if not already done since last release).

Reply:
@mickaelistria I (incorrectly by the sounds of things) assumed you'd want to keep the versions of all the bundles in step. I can change just the bundle I am modifying.

Reply:
@mickaelistria Understood, I'll update that.